src/test/java/life/qbic/openbis/openbisclient/TestOpenBisClient.java

```diff
@@ -124,11 +124,10 @@ public void testListExperiments() {
   }
```

Comment:
Is here somewhere an option to see if a sessionToken is valid? I guess aside from being undefined it could not exists. But then again it changes all the time...

Reply:
I have added an additional line which checks if the token is valid